README.md

```diff
@@ -340,11 +346,8 @@ Describe your component here and give usage examples.
 ### Basic Usage
 
 ```jsx live
-<MyComponent>
-  Hello!
-</MyComponent>
+<MyComponent>Hello!</MyComponent>
```

Comment:
Cand we not change this? The way it was before looks more readable to me

Reply:
It seems to me that it is expected that the code that fits on one line will have this format